### PR TITLE
Add nearly 2k entities! 🎉 

### PR DESCRIPTION
**Pull request by patrickhulce**
Massive shoutout to @simonhearne who generously offered his list up publicly! 🎉 🙏 💯

### CODE REVIEW

File: bin/sql-dump-to-entities.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+const _ = require('lodash')
+const fs = require('fs')
+const path = require('path')
+const colors = require('colors')
```
Comment (paulirish) [inline bin/sql-dump-to-entities.js]: afaict you don't use this anymore.

Comment (patrickhulce) [reply bin/sql-dump-to-entities.js]: oh thanks :) yeah I did some more logging initially of the differences but resolved all those and didn't need it anymore

**Comment by soulgalore** [comment]
@patrickhulce could you please push this in a new npm release?  Would love to try out all the new entities :)

**Comment by patrickhulce** [comment]
Oh yeah sure was planning on doing it tomorrow, but just published 0.3.0 :)

**Comment by soulgalore** [comment]
Thanks :)

**Pull request merged**